eclass/systemd.eclass

```diff
@@ -222,7 +222,7 @@ systemd_install_serviced() {
 
 	(
 		insopts -m 0644
-		insinto /etc/systemd/system/"${service}".d
+		insinto "$(_systemd_unprefix systemd_get_systemunitdir)"/"${service}".d
```

Comment:
This is incorrect as well. The files are intended to be modified by the sysadmin.

Reply:
Many maintainers (including me) use this function as [patch mechanism](https://github.com/gentoo/gentoo/pull/33007/files#diff-53900fabe15fc262caa52995e3787f705fa59d07c28ae7930efa29ec835197a0R1).  Should there be another function for those who want to use it like that?

Reply:
Yes, you should use patches to patch stuff.

Reply:
@mgorny You know the best, sir. This PR is useless then, it seems. Closing...

Reply:
Per [bug 876658](https://bugs.gentoo.org/876658), I would accept a patch to add a new function to install distro-level overrides in /lib/systemd/system/foo.service.d and /usr/lib/systemd/user/foo.service.d.

Reply:
Thanks. Ideally we also add documentation to the existing (and new) function that explains that files are installed under /etc (and (/usr)/lib respective).